bedrock/mozorg/views.py

```diff
@@ -251,20 +251,8 @@ def home_tweets(locale):
     return []
 
 
-def home(request, template='mozorg/home/home.html'):
-    locale = l10n_utils.get_locale(request)
-
-    if lang_file_is_active('mozorg/home/index-2016', locale):
-        template = 'mozorg/home/home.html'
-    else:
-        template = 'mozorg/home/home-voices.html'
-
-    return l10n_utils.render(
-        request, template, {
-            'has_contribute': lang_file_is_active('mozorg/contribute'),
-            'tweets': home_tweets(locale),
-            'mobilizer_link': settings.MOBILIZER_LOCALE_LINK.get(
-                locale, settings.MOBILIZER_LOCALE_LINK['en-US'])})
+def home(request):
+    return l10n_utils.render(request, 'mozorg/home/home.html')
```

Comment:
I thought about removing this view and turning the URL back to a regular `page` rule, but that would mean updating all the internal home page links on other pages, of which there are a shit ton. I decided it wasn't worth the effort, but can be persuaded otherwise.